### PR TITLE
GSLUX-759: Fix metadata and legend missing for some layers

### PR DESCRIPTION
**Pull request by AlitaBernachot**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-759

### Description

- fix display metadata for layer (id 2760)
- fix legend
- add support for legend translation (when server returns html content with angular directive "translate")
- update fixtures with current prod content

### CODE REVIEW

File: src/composables/translateParser.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * Translate legacy _V3 text
+ * eg: from getLegendHtml, server returns html to be translated with angular directive "<h4 translate>Rigoles d&#39;érosion identifiées</h4>"
+ * @returns
+ */
+export function useTranslateParser() {
+  const { t } = useTranslation()
+
+  const translate = (htmlContent: string, options?: TOptions | string) => {
```
Comment (mki-c2c) [inline src/composables/translateParser.ts:12]: rename to luxTranslate ?

Is there any good practice in use of arrow functions vs. function declaration ?
Instinctively, I find a function declaration with keyword 'function' more readable inside a class-like structure like here.

File: src/composables/translateParser.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { TOptions } from 'i18next'
+import { useTranslation } from 'i18next-vue'
+
+/**
+ * Translate legacy _V3 text
+ * eg: from getLegendHtml, server returns html to be translated with angular directive "<h4 translate>Rigoles d&#39;érosion identifiées</h4>"
+ * @returns
+ */
+export function useTranslateParser() {
```
Comment (mki-c2c) [inline src/composables/translateParser.ts:9]: translation parser would be rather a service than a composable, no reactivity involved.

**Review by mki-c2c — APPROVED** [review]
LGTM did not test in v3 context yet, so there are the CORS limitations

File: cypress.config.ts [hunk 3]
```diff
@@ -2,8 +2,9 @@ import { defineConfig } from 'cypress'
 
 export default defineConfig({
   e2e: {
-    specPattern: 'cypress/e2e/**/*.{cy,spec}.{js,jsx,ts,tsx}',
     baseUrl: 'http://localhost:4173',
+    chromeWebSecurity: false,
```
[the remark this answers is not in the document]
Comment (AlitaBernachot) [reply cypress.config.ts:6]: > what's our cypress test strategy ? I usually run cypress tests with electron browser. 
No strategy, it should work for all browsers.

>Is there a reason for switching to chrome ?
Yes, for this settings `chromeWebSecurity: false,` that allows the e2e tests to bypass CORS policies.

File: src/composables/translateParser.ts (hunk 1 above)
Comment (AlitaBernachot) [reply src/composables/translateParser.ts:12]: > Is there any good practice in use of arrow functions vs. function declaration ? Instinctively, I find a function declaration with keyword 'function' more readable inside a class-like structure like here.

There are no good practices, arrow functions may be helpful when using `this` context, removing the confusion to what it refers at the moment the function is called. Arrow function have been introduced in es6, so it is a modern way to declare function.

File: src/composables/translateParser.ts (hunk 2 above)
Comment (AlitaBernachot) [reply src/composables/translateParser.ts:9]: It is a composable because it uses i18next's `useTranslation().`

**Pull request merged**